pkg/config/loader.go

```diff
@@ -28,13 +28,14 @@ const (
 	// Default Configuration for Pod ENI resource type
 	PodENIDefaultWorker = 30
 
-	// Default Configuration for IPv4 resource type
-	IPv4DefaultWorker  = 2
-	IPv4DefaultWPSize  = 3
-	IPv4DefaultMaxDev  = 1
-	IPv4DefaultResSize = 0
-
-	// Default Configuration for IPv4 prefix resource type
+	// Default Windows Configuration for IPv4 resource type
+	IPv4DefaultWinWorkerCount  = 2
+	IPv4DefaultWinWarmIPTarget = 1
```

Comment:
@haouc @jiechen0826 Any thoughts on optimal default values for `minIPTarget` and `warmIPTarget` for secondary IP mode? I am currently leaning towards having the same default values as Linux uses for consistency.

Reply:
The thing is Windows customers have been getting 3 warm IPs as default in the past, so I would argue for `warmIPTarget`=3 for consistency.

Reply:
I updated to  `warmIPTarget=3`